src/uct/tcp/sockcm/sockcm_cm.h

```diff
@@ -0,0 +1,25 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#include <uct/base/uct_cm.h>
+
+#define UCT_SOCKCM_CM_PRIV_DATA_LEN  1024  /** Closest power of two to common MTU value in TCP (1500) */
+
+/**
+ * A sockcm connection manager
+ */
+typedef struct uct_sockcm_cm {
+    uct_cm_t super;
+} uct_sockcm_cm_t;
+
+typedef struct uct_sockcm_priv_data_hdr {
+    uint8_t  length;     /* length of the private data */
```

Comment:
indent on same column as uct_sockcm_cm_t::super

Reply:
it already is... super and length are on column 14.
am i missing something?

Reply:
ok then